.cloudbuild.yaml

```diff
@@ -97,8 +97,12 @@ steps:
   id: 'buildDeployer'
 - name: 'gcr.io/cloud-builders/docker'
   entrypoint: '/bin/bash'
-  args: ['-c', 'cd /workspace/components/kubeflow/launcher && ./build_image.sh -p $PROJECT_ID -t $COMMIT_SHA']
```

Comment:
Could we not build the container automatically for now? It requires a complicated review process on our side.

Reply:
I just rename launcher (in fact, it only launches tfjob) to tf-launcher since we can add more launchers (saying pytorch job launcher, studyjob launcher and so on)

Reply:
Thank you @hougangliu. Could we do the renaming in a separate PR? There are several dependencies on this tfjob component so I would like to separate the renaming into another PR in case we need to rollback.